.github/workflows/publish-rc.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Publish Release Candidate'
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/package.json'
+
+jobs:
+  publish-rc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+
+      - name: Install dependencies and publish
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+          registry-url: 'https://registry.npmjs.org'
+      - run: yarn && yarn publish:next
+        env:
+          YARN_NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTOMATION_FROM_STEPHEN }}
+
+      - name: Branch protection OFF
+        run: |
+          curl --request PUT \
+          --url 'https://api.github.com/repos/${{ github.repository }}/branches/main/protection' \
+          --header 'Authorization: token ${{ secrets.PAT_CHANGE_BRANCH_PROTECTION_RULES_FROM_STEPHEN }}' \
+          --data '{"required_status_checks":null,"enforce_admins":false,"required_pull_request_reviews":null,"restrictions":null}'
```

Comment:
Curious about how this endpoint works. If I was using the GUI manually I can disable `enforce_admins` to ignore the rules as an admin, I guess that wouldn't work here which is why we need to explicitly disable all of them?

Just wondering if there's a way to auth the push so that we can change fewer rules, or is it a requirement to pass the entire object?

Reply:
Hmmm oh yeah, you're right. I can just `enforce_admins: false` and `enforce_admins: true` instead

Reply:
That's a shame... looks like the endpoint makes the other fields required: 
![image](https://user-images.githubusercontent.com/175330/132500028-41b7fe44-931d-4163-a766-dec367dab831.png)

Reply:
boooo 👎

Reply:
Aw that's a shame. If it turns into a problem in the future perhaps there's a way to do `get` and restore them from that or something.

Reply:
Ooo, let me try that. Might not be too tricky 🤔

Reply:
Yeah.. if it looks remotely tricky then probably not worth spending the brain cycles on. Just a thought though.

Reply:
It's tricky... haha. We've decided to leave as is because we rarely change these rules, if ever.